raspberry_pi/app/localization/combined_detector.py

```diff
@@ -176,17 +176,17 @@ def analyze(self, req: Request) -> None:
             # Get BGR image for both detectors
             img_bgr = cast(Mat, np.frombuffer(buffer_rgb, dtype=np.uint8))
             img_bgr = img_bgr.reshape((self.height, self.width, 3))
-
+            img_display = img_bgr.copy()
```

Comment:
do you need this img_display thing?  i thought you only used this while you were trying to figure out how to do the color correction.

Reply:
ah, ok, got it.